### ISSUE TITLE
AttributeError: 'str' object has no attribute 'items' in go_enrichment (py2.7)

### ISSUE THREAD

**Issue by IvoLeist** [saezlab/pypath#29]
Dear Denes,

following your documentation on http://pypath.omnipathdb.org/index.html#example-3-pathways-annotations my code crashes again in example 4:

```
notch = pa.pathway_members('Notch', 'signalink')
notch.up()
enr = pa.go_enrichment(list(notch.up()))
```
runs into:

> AttributeError                            Traceback (most recent call last)
> <ipython-input-36-e7dcb82e4de1> in <module>()
>       5 notch = pa.pathway_members('Notch', 'signalink')
>       6 notch.up()
> ----> 7 enr =pa.go_enrichment(list(notch.up()))
> 
> /home/user/.local/lib/python2.7/site-packages/pypath/main.pyc in go_enrichment(self, proteins, aspect, alpha, correction_method, all_proteins)
>    6870         annotation = dict([(up, g)
>    6871                            for up, g in getattr(self.go[self.ncbi_tax_id],
> -> 6872                                                 iteritems(aspect.lower()))
>    6873                            if up in all_proteins])
>    6874         enr = go.GOEnrichmentSet(
> 
> /home/user/.local/lib/python2.7/site-packages/future/utils/__init__.pyc in iteritems(obj, **kwargs)
>     304     func = getattr(obj, "iteritems", None)
>     305     if not func:
> --> 306         func = obj.items
>     307     return func(**kwargs)
>     308 
> 
> AttributeError: 'str' object has no attribute 'items'

What i tried so far:
- checked for newest version of future/utils
```
-bash-4.2$ pip freeze | grep "future"
future==0.16.0
futures==3.2.0
```
- tried hardcoding it
pa.go_enrichment ('Q8NES3','Q9NYJ7') --> AttributeError: 'str' object has no attribute 'items'
pa.go_enrichment (u'Q8NES3',u'Q9NYJ7') --> AttributeError: 'unicode' object has no attribute 'items'

I guess it is once again a python2.7 specific error ?

Best,
Ivo

**Comment by Nic-Nic** [comment]
Dear Ivo,

I reproduced what you've done in Python 2.7 and I did not get any error (with PyPath 0.7.111).
Which version are you running? Do you still get the same error after update?

**Comment by IvoLeist** [comment]
Hello Nic-Nic with PyPath version: 0.7.111
using Python 2.7.5. I now get the following output when executing:

`notch = pa.pathway_members('Notch', 'signalink')
notch.up()
enr = pa.go_enrichment(list(notch.up()))
print enr` 

> `[<pypath.enrich.Enrichment object at 0x7f90fd120950>, ...shortened..., <pypath.enrich.Enrichment object at 0x7f90fd128dd0>] foo bar [6.466726301439505e-06,...shortened...,0.07938138362322154]
Traceback (most recent call last):
  File "omniPath_tutorial.py", line 79, in <module>
    print enr
  File "/home/user/.local/lib/python2.7/site-packages/pypath/go.py", line 165, in __str__
    self.top_terms(length=10, significant=True)]) + '\n'
  File "/home/user/.local/lib/python2.7/site-packages/pypath/enrich.py", line 127, in top_names
    args = get_args(locals())
NameError: global name 'get_args' is not defined

*I did not printed the "foo bar" in my script I guess it is a debug artefact in your script ?

Best,
Ivo
`

**Comment by Nic-Nic** [comment]
There was missing an import for that function, thanks for spotting the issue.
Will be solved once #61 is merged.
